### PR TITLE
PD-3682 - PA/Analytics - Footer Style

### PR DESCRIPTION
**Pull request by iartemiev**
## What's this PR do?
Fixes footer style in PA and Analytics

## Screenshots (if appropriate)
image

image

### CODE REVIEW

File: gulp/src/sass/partner_assessment/main.scss [hunk 1]
```diff
@@ -18,8 +18,10 @@
 @import "./components/_record_list_filter";
 @import "./components/_assessment-form";
 
-
 .partner-assessment-app {
+    background-color: white;
+    border: 1px solid #333;
+    padding-bottom: 15px;
```
Comment (aljordan82) [inline gulp/src/sass/partner_assessment/main.scss]: Is this padding needed? I think it should be fine without it.

Comment (aljordan82) [reply gulp/src/sass/partner_assessment/main.scss]: Also, I would probably remove border left and right.

Comment (iartemiev) [reply gulp/src/sass/partner_assessment/main.scss]: With padding:
image

Without padding:
image

Comment (iartemiev) [reply gulp/src/sass/partner_assessment/main.scss]: If you like the way that looks, I'm happy to remove it

Comment (aljordan82) [reply gulp/src/sass/partner_assessment/main.scss]: hmm? it looked like it has excessive padding on my machine?

**Pull request merged**